docs/mobile/exercises/README.md

```diff
@@ -0,0 +1,294 @@
+
+```yaml
+# print()
+- key: c11dbdf1-49c9-408a-a1e4-885431af2196
+  context: "`print()` outputs to the screen. Whatever you put between the parentheses will be output! E.g. `print(5)` will output `5`!"
+  question: "" # this is the default
+  code: "print(42)"
+  output: "42"
+  distractor_output:
+    - option: "4"
+      feedback: "`print()` will output all of whatever is between the parentheses, not just the first part."
+    - option: "2"
+      feedback: "`print()` will output all of whatever is between the parentheses, not just the last part."
+```
+
+Let's dig into each of the blocks...
+
+## topic
+
+```yaml
+# print()
+```
+
+Even though it's not part of the exercise itself, it's a very good idea to
+comment your exercises so you can quickly remind yourself what each exercise is
+designed to teach or test. In fact, it can be helpful to start by outlining a
+whole lesson with a scaffolding of comments before expanding the exercises.
+
+## key
+
+```yaml
+- key: c11dbdf1-49c9-408a-a1e4-885431af2196
+```
+
+This is a unique UUID for the exercise.
+
+## context
+
+```yaml
+  context: "`print()` outputs to the screen. Whatever you put between the parentheses will be output! E.g. `print(5)` will output `5`!"
+```
+
+This is where you can provide exposition necessary to introduce a new
+concept. It should be kept as minimal as possible (no longer than 150
+characters). It's good to shoot for a lighter, more informal feel here than on
+desktop. This field supports markdown elements like `inline code` and
+**emboldening**. Emojis are not just supported, they're encouraged!
+
+## question
+
+```yaml
+  question: "" # this is the default
+```
+
+This field is always optional. It will override the default question for the
+relevant exercise type. Don't change this unless you absolutely need to for an
+exercise to make sense.
+
+## code
+
+```yaml
+  code: "print(42)"
+```
+
+This is the code! Anything you here should be syntactically correct, executable
+code. The format of the code block can vary slightly for different exercise
+types, for example reorder and tap exercises. See the documentation for those
+exercise types to get the details.
+
+## output
+
+```yaml
+  output: "42"
+```
+
+This is the output generated by the code in a code block. That means anything
+that would get printed to the console in an interactive Python or R terminal
+session.
+
+## distractors
+
+```yaml
+  distractor_output:
+    - option: "4"
+      feedback: "`print()` will output all of whatever is between the parentheses, not just the first part."
+    - option: "2"
+      feedback: "`print()` will output all of whatever is between the parentheses, not just the last part."
+```
+
+The rendered exercise type depends largely on the distractors block. In the
+example above, the use of the `distractor_output` field indicates this exercise
+will be a select output exercise.
+
+Other distractor types will result in different exercise types:
```

Comment:
I don't think different exercise types have been discussed yet, which makes this confusing.